src/azure-cli-core/azure/cli/core/auth/util.py

```diff
@@ -158,7 +158,7 @@ def build_sdk_access_token(token_entry):
     # Importing azure.core.credentials.AccessToken is expensive.
     # This can slow down commands that doesn't need azure.core, like `az account get-access-token`.
     # So We define our own AccessToken.
-    return AccessToken(token_entry["access_token"], request_time + token_entry["expires_in"])
+    return AccessToken(token_entry["access_token"], str(request_time) + token_entry["expires_in"])
```

Comment:
Casting `request_time` from int to str totally changes the program logic, because calling `+` on 2 strings concatenates them, which is certainly not what we want here. 

Instead, we need to calculate the `expires_on` by adding the int `expires_in` to the current int epoch time.